.github/workflows/validation-jobs.yml

```diff
@@ -0,0 +1,69 @@
+name: validation jobs
+
+on:
+  push:
+     branches:
+       - staging
+       - trying
+       - main
+
+env:
+  CARGO_TERM_COLOR: always
+
+jobs:
+  run-examples-on-wasm:
+    runs-on: ubuntu-latest
+    timeout-minutes: 30
+    steps:
+      - uses: actions/checkout@v3
+
+      - uses: actions-rs/toolchain@v1
+        with:
+          toolchain: stable
+          target: wasm32-unknown-unknown
+          override: true
+
+      - uses: actions/cache@v3
+        with:
+          path: |
+            ~/.cargo/bin/
+            ~/.cargo/registry/index/
+            ~/.cargo/registry/cache/
+            ~/.cargo/git/db/
+            target/
+          key: ${{ runner.os }}-wasm-run-examples-${{ hashFiles('**/Cargo.toml') }}
+
+      - name: install xvfb, llvmpipe and lavapipe
+        run: |
+          sudo apt-get update -y -qq
+          sudo add-apt-repository ppa:oibaf/graphics-drivers -y
+          sudo apt-get update
+          sudo apt install -y xvfb libegl1-mesa libgl1-mesa-dri libxcb-xfixes0-dev mesa-vulkan-drivers
+
+      - name: Install wasm-bindgen
+        run: cargo install --force wasm-bindgen-cli
+
+      - name: Run examples
+        shell: bash
+        run: |
+          # start a webserver
+          python3 -m http.server --directory examples/wasm &
```

Comment:
I'm glad this doesn't block the test from completing. 
My naïve assumption would be that it would

I still don't really like this though, it *feels* like we're assuming something we shouldn't be.

Reply:
this is approved by GitHub staff in the forums: https://github.community/t/how-do-i-run-commands-concurrently/16679/2